torch/distributed/_spmd/distribute.py

```diff
@@ -373,8 +389,8 @@ def _rebuild_graph(
                         dtn, lambda n: value_remap[n]
                     )
 
-    gm.graph.lint()
     gm.graph.eliminate_dead_code()
+    gm.graph.lint()
```

Comment:
The call to `lint` is superfluous as the first thing `eliminate_dead_code` does is call `lint`.

Reply:
Thanks, will remove in a followup PR. Wanted to avoid trigger CI again. :)